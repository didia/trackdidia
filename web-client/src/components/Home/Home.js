import React from 'react';

let defaultStyles = {
    width: '100%',
    display: 'flex',
    padding: '3vh 3vw',
    alignItems: 'center',
    justifyContent: 'center'
};
const Home = () => {
    return (
        <div className="Home" style={defaultStyles}>
            Home of the future Trackdidia App!
        </div>
    )
}
export default Home;
