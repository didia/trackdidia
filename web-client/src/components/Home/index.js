/**
 * Created by didia on 16-10-15.
 */
import Home from './Home';

export default Home;
