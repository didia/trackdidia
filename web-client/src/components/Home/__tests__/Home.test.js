/**
 * Created by didia on 16-10-15.
 */
import React from 'react';
import ReactDOM from 'react-dom';
import Home from '../Home';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Home />, div);
});

it('has not unexpectedly changed', () => {
    const component = renderer.create(
        <Home />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
