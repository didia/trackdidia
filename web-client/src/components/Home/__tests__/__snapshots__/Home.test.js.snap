exports[`test has not unexpectedly changed 1`] = `
<div
  className="Home"
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "center",
      "padding": "3vh 3vw",
      "width": "100%"
    }
  }>
  Home of the future Trackdidia App!
</div>
`;
